Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1532,4 +1536,98 @@ namespace Babylon
     {
         return Napi::Value::From(info.Env(), m_graphicsImpl.GetHardwareScalingLevel());
     }
+
+    Napi::Value NativeEngine::CreateImageBitmap(const Napi::CallbackInfo& info)
+    {
+        auto env = info.Env();
+        if (!info[0].IsArray())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap first parameter must be an array.");
+        }
+        const auto parameters = info[0].As<Napi::Array>();
+        if (!parameters.Length())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap parameters are empty.");
+        }
+
+        const auto blob = parameters[0u].As<Napi::Array>();
```

Comment:
Should we be checking that this is actually an array (like we do for `info[0]`)?

Reply:
Yes, I'm adding a check.